etc/config-template.toml

```diff
@@ -158,6 +158,15 @@
 ## Otherwise, the OOM problem might occur in TiKV.
 # capacity = "1GB"
 
+## Optional memory allocator to allocate shared block cache.
+## If set jemalloc nodump allocator, block cache will be excluded from core dump, which is useful to
+## reduce core file size. This allocator is available only if "jemalloc" feature is enabled.
+##
+## Default: not set, use default allocator of RocksDB
+## jemalloc nodump allocator: "jemalloc-nodump"
+##
+#  memory-allocator: "jemalloc-nodump"
```

Comment:
I think we can use it as default now, we don't need to add a configuration in the template here.

Reply:
Okay, just thought maybe we will support more allocator in the future.

Reply:
Let's just have a `dump-block-cache: true|false` option, which is easier to understand.